src/main/webapp/components/workspaces/filter-workspaces.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import { List, ListItem, ListItemText, Menu, MenuItem } from '@material-ui/core'
```

Comment:
Can you import these individually rather than deconstructing them? We ran into some build performance regressions a while back due to the overhead added from this. https://github.com/connexta/revelio/pull/131